core/model/src/payment.rs

```diff
@@ -69,37 +70,62 @@ pub mod local {
     }
 
     #[derive(Clone, Debug, Serialize, Deserialize)]
-    pub struct Init {
-        pub identity: NodeId,
-        pub provider: bool,
-        pub requestor: bool,
+    pub struct RegisterAccount {
+        pub driver: String,
+        pub platform: String,
```

Comment:
Same here, both `driver` and `platform` are redundant.

Reply:
IMHO only `driver` is necessary.

Reply:
I believe the `platform` parameter has to stay here. Payment service has no other way to know which driver handles which platform.